lib-esm/constants.js

```diff
@@ -19,6 +19,8 @@ const whiteSpace = system({
 });
 const TYPOGRAPHY = compose(styledSystem.typography, whiteSpace);
 // Border props
-compose(styledSystem.border, styledSystem.shadow);
+const BORDER = compose(styledSystem.border, styledSystem.shadow);
+// Layout props
+const LAYOUT = styledSystem.layout;
 
-export { COMMON, TYPOGRAPHY, get };
+export { BORDER, COMMON, LAYOUT, TYPOGRAPHY, get };
```

Comment:
i'm definitely missing something because it looks like border and layout were not exported till now? is it because they were tree shaken?

Reply:
I think so, it seemed like they aren't re-exported through `src/index.ts` and were removed based on usage in the codebase

Reply:
yay it's fixed!